pennylane/ops/qubit/special_unitary.py

```diff
@@ -124,21 +125,31 @@ def pauli_basis_strings(num_wires):
     return ["".join(letters) for letters in product(_pauli_letters, repeat=num_wires)][1:]
 
 
+def _detach(array, interface):
+    """Detach an array from its trace and return just its numerical values."""
+    if interface == "jax":
+        import jax
+
+        return jax.lax.stop_gradient(array)
+    return qml.math.convert_like(qml.math.to_numpy(array), array)
+
+
 class SpecialUnitary(Operation):
     r"""Gate from the group :math:`SU(N)` with :math:`N=2^n` for :math:`n` qubits.
 
     We use the following parametrization of a special unitary operator:
 
     .. math::
 
-        U(\theta) &= e^{A(\theta)}\\
-        A(\theta) &= \sum_{m=1}^d i \theta_m P_m\\
-        P_m &\in \{I, X, Y, Z\}^{\otimes n} \setminus \{I^{\otimes n}\}
+        U(\bm{\theta}) &= \exp\{A(\bm{\theta})\}\\
+        A(\bm{\theta}) &= \sum_{m=1}^d i \theta_m P_m\\
+        P_m &\in {I, X, Y, Z}^{\otimes n} \setminus \{I^{\otimes n}\}
```

Comment:
It does not render correctly the bracket needs to appear

Reply:
Thanks!